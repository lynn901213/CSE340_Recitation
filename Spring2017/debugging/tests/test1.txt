hello, world!
this is a test
1234 is a number
