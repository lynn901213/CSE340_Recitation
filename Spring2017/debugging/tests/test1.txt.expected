1234 is a number
this is a test
hello, world!
